### PR TITLE
Document $suffix argument to tempnam()

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]
I guess it must be merged in 3.4, but we take care of it 👍🏻👌🏻

File: components/filesystem.rst [hunk 1]
```diff
@@ -284,6 +284,8 @@ tempnam
 
     // returns a path like : /tmp/prefix_wyjgtF
     $filesystem->tempnam('/tmp', 'prefix_');
+    // returns a path like : /tmp/prefix_wyjgtF.png
+    $filesystem->tempnam('/tmp', 'prefix_', '.png');
```
Comment (pierredup) [inline components/filesystem.rst]: I think you can add a block below specifying the version this was introduced in

```
.. versionadded:: 4.4

    The option to set a suffix in  ``tempnam()`` was introduced in Symfony 4.4.
```

Comment (OskarStark) [reply components/filesystem.rst]: Yes please 👍🏻

**Comment by jdufresne** [comment]
Thanks. I added the `versionadded`.

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Pull request merged**